doc/installation/iOS.md

```diff
@@ -1,80 +1,169 @@
 ## Installation - iOS
 
-After setting up your [environment][1], you can follow the steps below to run the **pass Culture** mobile application on iOS.
+**After setting up your [environment][1]**, you can follow the steps below to run the **pass Culture** mobile application on iOS.
 
 ### Further installation process
 
-To install iOS modules (pods), we use CocoaPods (dependency manager). It is built with Ruby, so make sure it is installed on your computer.
+If you encounter errors during this setup, refer to the Troubleshooting section at the end of this document.
 
-Install RVM (Ruby Version Manager) with this command :
+<br />
 
-```sh
-curl -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer | bash -s stable
-```
+- **install `node`**
 
-Install a version of Ruby that's 2.7.5 or higher (version of Ruby is indicated in the Gemfile on the root of the repo) with :
+  - with NVM to have the right version
 
-```sh
-rvm install "version"
-rvm use "version"
-```
+  ```sh
+  brew install nvm
+  nvm install
+  nvm use
+  ```
+
+  - **AND** with Brew to have XCode build's steps (`Bundle React Native code and images` and `Upload Debug Symbols to Sentry`) working
+
+  ```sh
+  brew install node
+  ```
+
+  - then we will use node v16 and put it by default:
+
+  ```
+  nvm install v16
+  nvm use v16
+  nvm alias default v16
+  ```
+
+<br />
+
+- **install RVM (Ruby Version Manager) and Ruby**
+
+  To install iOS modules (pods), we use CocoaPods (dependency manager). It is built with Ruby, so make sure it is installed on your computer, otherwise :
+
+  <br />
+
+  - install RVM with :
 
-We use [bundler][2] to install our Ruby gems. You can install it with `gem install bundler`.
+  ```sh
+  curl -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer | bash -s stable
+  ```
+
+  <br />
+
+  - install a version of Ruby that's 2.7.5 or higher (version of Ruby is indicated in the [Gemfile](/Gemfile) on the root of the repo) with :
+
+  ```sh
+  rvm install "version"
+  rvm use "version"
+  ```
+
+<br />
 
-Now you can install CocoaPods with `bundle install`. This will install the gems.
+- **install bundler**
 
-To install the pods, and create the Pods folder, run:
+  - We use [bundler][2] to install our Ruby gems, install it with :
 
-- `bundle exec pod install` inside the folder `ios`
-- you may be asked to run `bundle exec pod repo update`
+  ```sh
+  gem install bundler
+  ```
 
-If `bundle exec pod install` didn't work, check in Xcode -> Settings -> Locations if you have a Command line tools installed, then revalidate the Command line tools by clicking it.
+  <br />
+
+  - Now you can install CocoaPods, that will install the gems, with :
+
+  ```sh
+  bundle install
+  ```
+
+  <br />
+
+- **install the pods, and create the Pods folder**
+
+  - inside the folder `ios` :
+
+  ```sh
+  bundle exec pod install
+  ```
+
+  <br />
+
+  - you may be asked to run :
+
+  ```sh
+  bundle exec pod repo update
+  ```
```

Comment:
suggestion: pour éviter toute confusion, je recommande de changer de répertoire pas avec du texte mais dans la commande directement

~~~suggestion
  ```sh
  pushd ios
  bundle exec pod install
  popd
  ```

  you may be asked to run :

  ```sh
  pushd ios
  bundle exec pod repo update
  popd
  ```
~~~